docs/introduction/infrastructure.md

```diff
@@ -1,161 +1,104 @@
-the access to those resources, and even help you find the best one for your particular
-case (e.g. by price or by performance). 
+The Inductiva API acts as an abstraction layer that enables you to access a wide 
+array of computational resources across different providers through a unified 
+interface called the [Command Line Interface (CLI)](). The API's key role is to 
+assist you in identifying the resources best suited to your needs, whether you're prioritizing cost-efficiency or performance.
 
-Inductiva API is being designed to allow using compute resources such as:
+These resources could be from cloud providers, bare-metal hardware rentals, standard 
+high-performance computing (HPC) solutions commonly used in academia, or even on-premise 
+hardware for those with their own computing infrastructure. The API serves as a unifying interface atop all these varied resources, allowing you to tap into computational solutions of varying scales, prices, and performance levels through straightforward Python 
+scripting from your laptop.
 
-- fully working VMs provided by standard Cloud Providers;
-- hardware rented by “bare metal” providers;
-- standard HPC solutions (typically available to academic users);
-- on-premise hardware (for those who invested in owning their own compute solutions).
+From the server side, Inductiva manages your computational workload — be it one 
+or several simulations. It allocates this workload to the appropriate computational 
+resource available to you, handles the orchestration of the simulation, and then 
+returns the results back to you.
 
-Inductiva will function as a uniformizing layer on top of these heterogeneous resources
-from different providers, allowing users to access computational solutions with different
-levels of scale, price, performance and flexibility, just by running simple python
-scripts from their laptop.
+## Available Computational Resources
 
-## Available Computational Options
-As of version 0.4, Inductiva is able to send computational loads to Google Cloud
-Platform (GCP). Therefore, the simulations you start using the API will run on one
-VM or several VMs (if you start an MPICluster) that live on GCP. We make available
-VMs of the following families:
+In the current release, version 0.4, Inductiva supports dispatching computational 
+workloads to the Google Cloud Platform (GCP). This means that the simulations initiated through our API are executed on one or more virtual machines (VMs) hosted on GCP. 
 
-**Compute Optimized:**
+The VM families made available by Inductiva on Google Cloud
+Platform (GCP) include:
 
-- [C2](https://cloud.google.com/compute/docs/compute-optimized-machines#c2_machine_types)
-- [C2D](https://cloud.google.com/compute/docs/compute-optimized-machines#c2d_series)
-- [H3](https://cloud.google.com/compute/docs/compute-optimized-machines#h3_series)
 
-**General Purpose:**
+````{eval-rst}
+.. tabs::
```

Comment:
not working:

![Screenshot 2024-02-21 at 10 46 27](https://github.com/inductiva/inductiva/assets/7538022/2abc4ca6-e87e-4755-96c6-57267709c84d)